### PR TITLE
[BEAM-4706] BigQueryTornadoesIT cannot be run using integrationTest and performanceTest tasks

### PR DESCRIPTION
**Pull request by kkucharc**
In order to give the possibility to run IT tests from `beam-examples`, added the integration tests configuration to `beam-examples.gradle`. Additionally: fixed bug with passing pipeline options and added more specific documentation about running the test with gradle.

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ x ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | --- | --- | --- | ---

### CODE REVIEW

**Comment by kkucharc** [comment]
@lgajowy can you take a look at it, please?

File: examples/java/src/test/java/org/apache/beam/examples/cookbook/BigQueryTornadoesIT.java [hunk 1]
```diff
@@ -28,7 +28,23 @@
...
+ * 'BigQueryTornadoesIT' in a temporary location. Pass a temporary location in PipelineOptions:
+ *
+ * <pre>
+ *  ./gradlew integrationTest -p examples/java/ -DintegrationTestPipelineOptions='[
+ *  "--tempLocation=gs://your-location/"]'
+ *  --tests org.apache.beam.examples.cookbook.BigQueryTornadoesIT
+ *  -DintegrationTestRunner=direct
+ * </pre>
+ *
+ * Check {@link org.apache.beam.examples.cookbook.BigQueryTornadoes} form more configuration options
```
Comment (lgajowy) [inline examples/java/src/test/java/org/apache/beam/examples/cookbook/BigQueryTornadoesIT.java]: Add <p> before `Check`

File: examples/java/src/test/java/org/apache/beam/examples/cookbook/BigQueryTornadoesIT.java [hunk 2]
```diff
@@ -28,7 +28,23 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/** End-to-end tests of BigQueryTornadoes. */
+/**
+ * An end-to-end tests of {@link org.apache.beam.examples.cookbook.BigQueryTornadoes}.
+ *
+ * <p>This test uses the data from an external source defined in {@link
```
Comment (lgajowy) [inline examples/java/src/test/java/org/apache/beam/examples/cookbook/BigQueryTornadoesIT.java]: Temp location is only a place where temporary files are stored by the runner. I'm not sure that it has to be the same location as the BigQuery project. I'd skip the part where we mention `tempLocation` (it's described clearly in `PipelineOptions` interface), leaving it only in the running example and write something like this: 

```
This test reads the public samples of weather data from BigQuery, counts the number of tornadoes that occur 
in each month, and writes the results to BigQuery. It requires "BigQueryTornadoesIT" BigQuery dataset to be
created before running.

Running instructions:

<pre> ...
```

File: examples/java/src/test/java/org/apache/beam/examples/cookbook/BigQueryTornadoesIT.java [hunk 3]
```diff
@@ -28,7 +28,23 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/** End-to-end tests of BigQueryTornadoes. */
+/**
+ * An end-to-end tests of {@link org.apache.beam.examples.cookbook.BigQueryTornadoes}.
```
[the remark this answers is not in the document]
Comment (kkucharc) [reply examples/java/src/test/java/org/apache/beam/examples/cookbook/BigQueryTornadoesIT.java]: Ok, changed.

File: examples/java/src/test/java/org/apache/beam/examples/cookbook/BigQueryTornadoesIT.java (hunk 1 above)
Comment (kkucharc) [reply examples/java/src/test/java/org/apache/beam/examples/cookbook/BigQueryTornadoesIT.java]: Sure.

File: examples/java/src/test/java/org/apache/beam/examples/cookbook/BigQueryTornadoesIT.java (hunk 2 above)
Comment (kkucharc) [reply examples/java/src/test/java/org/apache/beam/examples/cookbook/BigQueryTornadoesIT.java]: Ok, I checked it and changed.

**Comment by lgajowy** [comment]
LGTM. Waiting for the PreCommits to be green and then merging.

Thanks!

**Pull request merged**